src/data/pages/home/index.js

```diff
@@ -1001,6 +1001,136 @@ const data = {
       items: videoItems,
     },
   },
+
+  'tw-cn': {
+    hero: {
+      title: '對核心進行動態編程，以實現高效的網路、可觀察性、追蹤和安全性。',
+      blackButtonTitle: '項目概覽',
+      blackButtonUrl: '/tw-cn/applications',
+      yellowButtonTitle: '什麼是eBPF？',
+      yellowButtonUrl: '/tw-cn/what-is-ebpf',
+      altImage: 'eBPF diagram',
+      items: [
+        '程序經過驗證，確保安全執行',
+        '在核心中任何位置勾點(hook)進行功能的修改',
+        '即時(JIT)編譯器可實現接近本機的執行速度',
+        '可在運行時增加作業系統功能',
+      ],
+    },
+    caseStudies: {
+      title: '每個行業的組織在生產中都使用 eBPF',
+      linkText: '更多案例研究',
+      linkUrl: '/tw-cn/case-studies/',
+      items: caseStudiesData['tw-cn'].slice(0, 6),
+    },
+    whyEbpf: {
+      title: '為什麼使用eBPF？',
+      linkTitle: '什麼是eBPF？',
+      linkUrl: '/zh-cn/what-is-ebpf/',
+      items: [
+        {
+          title: '效能',
+          icon: 'performance',
+          description: 'eBPF 透過即時編譯以及直接在核心中運行，大大提高了處理能力。',
+        },
+        {
+          title: '安全性',
+          icon: 'security',
+          description: 'eBPF 程式經過驗證不會導致核心崩潰，並且只能由特權使用者修改。',
+        },
+        {
+          title: '靈活性',
+          icon: 'flexibility',
+          description: '功能和用例的修改與添加可以即時加載到核心並生效，而無需重新啟動或打補丁。',
+        },
+      ],
+    },
+    film: {
+      label: 'premiere',
+      title: '釋放核心潛力',
+      videoId: 'Wb_vD3XZYOA',
+      description: 'eBPF 紀錄片深入探討了 eBPF 的起源，並展示了這項改變產業的技術的故事、挑戰和回報。 您將聽到開源世界中最傑出的人士的演講，包括來自Meta、Intel、Isovalent、Google、Red Hat 和Netflix 的關鍵合作夥伴，他們幫助塑造和構建的工具推動了 eBPF 成功和採用。',
+      buttonText: 'eBPF 紀錄片',
+      buttonUrl: 'https://ebpfdocumentary.com/',
+    },
+    testimonials: {
+      items: [
+        {
+          logo: 'azure',
+          logoWidth: 134,
+          company: 'Microsoft Azure',
+          review: 'eBPF 帶來了新一代的工具，使開發人員能夠輕鬆地診斷問題、快速地創新以及擴展作業系統功能。',
+          author: 'Mark Russinovich',
+          role: 'Chief Technology Officer at Microsoft Azure, 2021',
+        },
+        {
+          logo: 'google',
+          logoWidth: '104',
```

Comment:
There should be a number here instead of a string. Could you update the value to a number?